### ISSUE TITLE
Internal Server Error if no external system id is specified

### ISSUE THREAD

**Issue by shaburov** [reportportal/reportportal#251]
Class `Issue.ExternalSystemIssue`
An error occurs if the request does not specify `systemId`

![image](https://user-images.githubusercontent.com/15269131/32018504-c0762fd8-b9d2-11e7-91ec-bd4522a7f03c.png)

```
Caused by: com.epam.reportportal.exception.ReportPortalException: Report Portal returned error
Status code: 500
Status message: Internal Server Error
Error Message: Unclassified error [The given id must not be null!]
Error Type: UNCLASSIFIED_ERROR
```

**Comment by avarabyeu** [comment]
@shaburov which exactly resource do you call?

**Comment by DzmitryHumianiuk** [comment]
@shaburov up

**Comment by shaburov** [comment]
If an error occurs, listing the identifiers would be a good solution

![image](https://user-images.githubusercontent.com/15269131/32542312-39304318-c483-11e7-801a-e9e48c7d6f9c.png)

**Comment by avarabyeu** [comment]
@shaburov 
```
Caused by: com.epam.reportportal.exception.ReportPortalException: Report Portal returned error
Status code: 500
Status message: Internal Server Error
Error Message: Unclassified error [The given id must not be null!]
Error Type: UNCLASSIFIED_ERROR
```
what operation causes this error?

**Comment by avarabyeu** [comment]
is it test item finish?

**Comment by shaburov** [comment]
@avarabyeu yes ^

**Comment by DzmitryHumianiuk** [comment]
@pbortnik up

**Comment by avarabyeu** [comment]
@evjlobanova plz add to 4.2 backlog. This should be fixed.

**Comment by e-olga** [comment]
The issue is fixed and will be available in release version RP 4.2
